### ISSUE TITLE
building the project

### ISSUE THREAD

**Issue by mandelyoni** [uzh-rpg/rpg_public_dronet#24]
Hi Antonio,

We're following your instructions, and when building  dronet_control, we get the following:

yoni@yoni-ThinkPad-E470:~/bebop_ws/dronet/dronet_control$ catkin build  --this
--------------------------------------------------------
Profile:                     default
Extending:          [cached] /opt/ros/kinetic
Workspace:                   /home/yoni/bebop_ws
--------------------------------------------------------
Source Space:       [exists] /home/yoni/bebop_ws/src
Log Space:          [exists] /home/yoni/bebop_ws/logs
Build Space:        [exists] /home/yoni/bebop_ws/build
Devel Space:        [exists] /home/yoni/bebop_ws/devel
Install Space:      [unused] /home/yoni/bebop_ws/install
DESTDIR:            [unused] None
--------------------------------------------------------
Devel Space Layout:          linked
Install Space Layout:        None
--------------------------------------------------------
Additional CMake Args:       None
Additional Make Args:        None
Additional catkin Make Args: None
Internal Make Job Server:    True
Cache Job Environments:      False
--------------------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
--------------------------------------------------------
Workspace configuration appears valid.
--------------------------------------------------------
[build] Found '5' packages in 0.0 seconds.                                                                                                                      
[build] Given package 'dronet_control' is not in the workspace

Is    /dronet/dronet_control   supposed to be a package  inside the main workspace src folder?

Thanks,  Yoni

**Comment by antonilo** [comment]
Hi Yoni, I think that the package dronet control is not detected in your catkin workspace. Can you try to install the ros package "catkin_simple", source your workspace and retry?

**Comment by mandelyoni** [comment]
Yes, sure, I will do it, but if dronet_control is a ROS package, why don't you put it inside the main src folder?
Thank you

**Comment by mandelyoni** [comment]
Hi, still same behaviour.
If dronet_control is a ROS package, why don't you put it inside the main src folder?
Thank you


yoni@yoni-ThinkPad-E470:~/bebop_ws/dronet/dronet_control$ roscd catkin_simple
yoni@yoni-ThinkPad-E470:~/bebop_ws/src/catkin_simple$ cd ~/bebop_ws/dronet/dronet_control/
yoni@yoni-ThinkPad-E470:~/bebop_ws/dronet/dronet_control$ catkin build  --this
--------------------------------------------------------
Profile:                     default
Extending:          [cached] /opt/ros/kinetic
Workspace:                   /home/yoni/bebop_ws
--------------------------------------------------------
Source Space:       [exists] /home/yoni/bebop_ws/src
Log Space:          [exists] /home/yoni/bebop_ws/logs
Build Space:        [exists] /home/yoni/bebop_ws/build
Devel Space:        [exists] /home/yoni/bebop_ws/devel
Install Space:      [unused] /home/yoni/bebop_ws/install
DESTDIR:            [unused] None
--------------------------------------------------------
Devel Space Layout:          linked
Install Space Layout:        None
--------------------------------------------------------
Additional CMake Args:       None
Additional Make Args:        None
Additional catkin Make Args: None
Internal Make Job Server:    True
Cache Job Environments:      False
--------------------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
--------------------------------------------------------
Workspace configuration appears valid.
--------------------------------------------------------
[build] Found '6' packages in 0.0 seconds.                                                                                                                    
[build] Given package 'dronet_control' is not in the workspace

**Comment by IhabMohamed** [comment]
Hi @mandelyoni,

I have followed step #4, as mentioned in "https://github.com/uzh-rpg/rpg_public_dronet/tree/master/drone_control/dronet", except (catkin build --this). I just build it using: catkin build, and there was no error. Hope it will work for you as well.

Regards,
Ihab
